### DISCUSSION TITLE
Building WiXUI Extention

### DISCUSSION THREAD

**Discussion by chrpai** [wixtoolset/issues#7119] [Questions] [answered]
I cloned the WiX UI repos and hit a couple errors.  The first one was easy.. add /4.0.0-rc.1 to the SDK project reference.  The second one is C++ related which I don't speak very well.  It says that it can't find wcautil.h.  The file doesn't exist in the repos but it's referenced by precomp.h.

What prereq steps/dependencies am I missing?

**Comment by robmen** [comment, marked as answer]
Which "WiX UI repos" are that? If it's https://github.com/wixtoolset/UI.wixext, then that is very, very outdated (and will be deleted). All WiX v4 code is in https://github.com/wixtoolset/wix4. The UI extension is in `src/ext/UI` folder.

**Reply by chrpai** [reply]
Ah, yes. I assumed that you moved away from monolithic and that each of those wixext was the source for the nuget package.  I'll look at wix4 instead then.
